### PR TITLE
Refactor of NFTs API

### PR DESCRIPTION
**Pull request by aaitor**
- API refactor
- First refactor of NFTs API

## Description

Add a description of your changes here.

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

#### Funny gif

Put a link of a funny gif inside the parenthesis-->

### CODE REVIEW

File: integration/config.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { LoggerInstance, LogLevel, makeAccounts } from '../src/utils'
 
 LoggerInstance.setLevel(LogLevel.Error)
 
+const logLevel = Number(process.env['LOG_LEVEL'])
```
Comment (r-marques) [inline integration/config.ts:6]: won't this fail if the LOG_LEVEL is not defined. Maybe we should add a default

File: src/keeper/Keeper.ts [hunk 2]
```diff
@@ -78,7 +78,7 @@ export class Keeper extends Instantiable {
                 // Main contracts
                 dispenser: undefined, // Optional
                 token: undefined, // Optional
-                nftUpgradeable: undefined, // Optional
+                // nftUpgradeable: undefined, // Optional
```
Comment (r-marques) [inline src/keeper/Keeper.ts:81]: cleanup?

File: src/keeper/Keeper.ts [hunk 3]
```diff
@@ -193,21 +194,13 @@ export class Keeper extends Instantiable {
...
-        } catch {
-            this.logger.debug('AaveCreditTemplate not available on this network.')
-        }
+        // try {
+        //     this.instances.aaveCreditTemplate = await AaveCreditTemplate.getInstance(
+        //         this.instantiableConfig
+        //     )
+        // } catch {
+        //     this.logger.debug('AaveCreditTemplate not available on this network.')
+        // }
```
Comment (r-marques) [inline src/keeper/Keeper.ts:203]: cleanup

File: src/keeper/Keeper.ts [hunk 4]
```diff
@@ -217,12 +210,22 @@ export class Keeper extends Instantiable {
...
+        // try {
+        //     console.log('\n')
+        //     this.instances.nftUpgradeable = await Nft1155Contract.getInstance(
+        //         this.instanceConfig
+        //     )
+        // } catch (error) {
+        //     this.logger.debug('NFT1155Upgradeable not available on this network.')
+        //     this.logger.debug((error as Error).message)
+        // }
+
```
Comment (r-marques) [inline src/keeper/Keeper.ts:222]: cleanup

**Comment by r-marques** [comment]
maybe NFTsBaseAPi Nfts1155Api and Nfts721Api should also go into `src/nevermined/nfts`?

**Review by r-marques — APPROVED** [review]

File: integration/config.ts (hunk 1 above)
Comment (aaitor) [reply integration/config.ts:6]: it uses `error` by default but adding a 1 (`warn`) as default value if `LOG_LEVEL` is not defined

File: src/keeper/Keeper.ts (hunk 2 above)
Comment (aaitor) [reply src/keeper/Keeper.ts:81]: :+1:

**Pull request merged**